src/api/src/routes/users/me/get.ts

```diff
@@ -22,16 +22,17 @@ export default class DiscordLoginRoute extends Route {
 			return next('uh oh, something broke');
 		}
 
-		const [user] = await this.sql<{ id: string; user_id: string; avatar: string; email: string; username: string }>`
-			select connections.id, user_id, avatar, email, username
-			from connections
-			join users
-			on connections.user_id = users.id
-			where access_token = ${req.oauth!.access_token}
-			and provider = ${req.oauth!.provider};
+		const [user] = await this.sql<{ avatar: string; email: string; username: string }>`
+			select id, avatar, email, username
+			from users
+			where id = ${req.oauth!.userId};
+		`;
+
+		const connections = await this.sql<{ }>`
+			select * from connections where user_id = ${req.oauth!.userId};
 		`;
 
 		req.statusCode = 200;
-		res.end(JSON.stringify({ token: req.oauth!.token, provider: req.oauth!.provider, user }));
+		res.end(JSON.stringify({ user, connections }));
```

Comment:
I looked into different possible scenarios:

- Hasura does not support cookies
- Hasura webhook mode does not pass cookies, neither in GET or POST mode

JWT "reading"/decoding does work, but we need to send the JWT token over the wire here for it to work, it does not need to include the `access_token` though.